release/cockpit-release.service

```diff
@@ -7,7 +7,7 @@ After=docker.service
 Environment="RELEASE_SINK=fedorapeople.org"
 Restart=always
 RestartSec=1800
-ExecStart=/bin/sh -xc "/usr/bin/docker run --name=cockpit-release --privileged --volume=/home/cockpit:/home/user:rw --volume=/home/cockpit/release:/build:rw --env=RELEASE_SINK=\"$RELEASE_SINK\" cockpit/infra-release -r https://github.com/cockpit-project/cockpit /build/bots/major-cockpit-release
+ExecStart=/bin/sh -xc "/usr/bin/docker run --name=cockpit-release --privileged --volume=/home/cockpit:/home/user:rw --volume=/home/cockpit/release:/build:rw --env=RELEASE_SINK=\"$RELEASE_SINK\" cockpit/release -r https://github.com/cockpit-project/cockpit /build/bots/major-cockpit-release
```

Comment:
This is missing a final `"`:
```
Oct 11 08:29:15 releasemachine1.novalocal systemd[1]: [/etc/systemd/system/cockpit-release.service:10] Unbalanced quoting in command line, ignoring: "/bin/sh -xc "/usr/bin/docker run --name=cockpit-release --privileged --volume=/home/cockpit:/home/user:rw --volume=/home/cockpit/release:/build:rw --env=RELEASE_SINK=\"$RELEASE_SINK\" cockpit/release -r https://github.com/cockpit-project/cockpit /build/bots/major-cockpit-release"
```
This is already in master, so supposedly we fixed that manually the last time around. But let's fix it properly while we are at it.